lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_object.rb

```diff
@@ -250,7 +250,7 @@ def process_assertions(message)
 
     def user_info_attributes(user)
       {'user' => user, 'tenant' => user.current_tenant, 'miq_group' => user.current_group}.each do |k, v|
-        value = MiqAeObject.convert_value_based_on_datatype(v.id, v.class.name)
+        value = MiqAeMethodService::MiqAeServiceModelBase.wrap_results(v)
```

Comment:
This solves the issue that the user object is got from DB with wrong current group in automate.

```MiqAeObject.convert_value_based_on_datatype``` calls ```find``` at service model layer which fetches the object from DB then calls ```MiqAeMethodService::MiqAeServiceModelBase.wrap_results``` on the object.

Since the user object is already available with the correct current group, we can call ```MiqAeMethodService::MiqAeServiceModelBase.wrap_results``` on it directly.

With this change, then https://github.com/ManageIQ/manageiq-content/pull/156 is not needed as the in-memory user object has the current group point to the group that the user is in when the request is made.